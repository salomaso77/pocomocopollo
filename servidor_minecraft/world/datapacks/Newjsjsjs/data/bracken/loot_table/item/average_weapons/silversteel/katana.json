{
	"pools": [{
		"rolls": 1,
		"entries": [{
			"type": "item",
			"name": "minecraft:netherite_sword",
			"functions": [{
				"function": "minecraft:set_name",
				"target": "item_name",
				"name": "Silversteel Katana"
			},{
				"function": "minecraft:set_lore",
				"entity": "this",
				"lore": [{
					"translate": "",
					"color": "blue",
					"italic": false
				},{
					"translate": "When in Main Hand:",
					"color": "gray",
					"italic": false
				},{
					"translate": "9 Attack Damage",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "1.6 Attack Speed",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "+2.5% Movement Speed",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "Projectile Deflection",
					"color": "aqua",
					"italic": false
				}],
				"mode": "insert"
			},{
				"function": "minecraft:set_components",
				"components": {
					"minecraft:item_model": "bracken:average_weapons/silversteel/katana",
					"minecraft:max_damage": 2150,
					"minecraft:enchantment_glint_override": false,
					"minecraft:blocks_attacks": {
          "block_delay_seconds": 0.25,
          "disable_cooldown_scale": 0.1,
          "item_damage": {
            "threshold": 0,
            "base": 4,
            "factor": 4
          },
          "block_sound": "minecraft:block.anvil.place",
          "disabled_sound": "minecraft:item.axe.scrape",
          "bypassed_by": "#bracken:bypasses_katana"
        }
				}
			},{
				"function": "set_enchantments",
				"enchantments": {
					"bracken:silvered_fix": 1
				}
			},{
				"function": "minecraft:set_attributes",
				"modifiers": [{
					"attribute": "minecraft:movement_speed",
					"id": "bracken:silversteel_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				},{
					"attribute": "minecraft:attack_speed",
					"id": "bracken:silversteel_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"attribute": "minecraft:attack_damage",
					"id": "bracken:silversteel_katana",
					"amount": 8,
					"operation": "add_value",
					"slot": "mainhand"
				}]
			},{
				"function": "minecraft:toggle_tooltips",
				"toggles": {
					"attribute_modifiers": false
				}
			},{
				"function": "set_damage",
				"damage": {
					"min": 0.48,
					"max": 0.93
				}
			}]
		}]
	}]
}