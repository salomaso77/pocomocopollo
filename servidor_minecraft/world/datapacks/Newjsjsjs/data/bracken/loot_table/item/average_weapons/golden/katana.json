{
	"pools": [{
		"rolls": 1,
		"entries": [{
			"type": "item",
			"name": "minecraft:golden_sword",
			"functions": [{
				"function": "minecraft:set_name",
				"target": "item_name",
				"name": "Golden Katana"
			},{
				"function": "minecraft:set_lore",
				"entity": "this",
				"lore": [{
					"translate": "",
					"color": "blue",
					"italic": false
				},{
					"translate": "When in Main Hand:",
					"color": "gray",
					"italic": false
				},{
					"translate": "4 Attack Damage",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "1.6 Attack Speed",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "+2.5% Movement Speed",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "Projectile Deflection",
					"color": "aqua",
					"italic": false
				}],
				"mode": "insert"
			},{
				"function": "minecraft:set_components",
				"components": {
					"minecraft:item_model": "bracken:average_weapons/golden/katana",
					"minecraft:blocks_attacks": {
          "block_delay_seconds": 0.25,
          "disable_cooldown_scale": 0.1,
          "item_damage": {
            "threshold": 0,
            "base": 4,
            "factor": 4
          },
          "block_sound": "minecraft:block.anvil.place",
          "disabled_sound": "minecraft:item.axe.scrape",
          "bypassed_by": "#bracken:bypasses_katana"
        }
				}
			},{
				"function": "minecraft:set_attributes",
				"modifiers": [{
					"attribute": "minecraft:attack_speed",
					"id": "bracken:golden_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"attribute": "minecraft:movement_speed",
					"id": "bracken:golden_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				},{
					"attribute": "minecraft:attack_damage",
					"id": "bracken:golden_katana",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				}]
			},{
				"function": "minecraft:toggle_tooltips",
				"toggles": {
					"attribute_modifiers": false
				}
			},{
				"function": "set_damage",
				"damage": {
					"min": 0.48,
					"max": 0.93
				}
			}]
		}]
	}]
}